5.3.a_perfect_dish_to_share.py

```diff
@@ -0,0 +1,33 @@
+# מנה מושלמת לחלוקה
+import math
+
+
+def is_perfect_num(num):
+    _sqrt = math.sqrt(num)
+
+    sum_dividers = 0
+
+    for i in range(2, math.ceil(_sqrt)):
```

Comment:
'i' I get the usage of 'i' in that case, should practice giving meaningful names